development/code/Game.ts

```diff
@@ -58,6 +59,7 @@ let isCastling = false;
 let isFriendlyFire = false;
 let isPieceKilled = false;
 let fellOffTheBoardPiece: Piece | undefined;
+let winner = '';
```

Comment:
It would make more sense for it to be of type `Player`. Something like `let winner: Player = undefined`